eea/api/dataconnector/upgrades/to6.py

```diff
@@ -0,0 +1,12 @@
+""" upgrade to 6.0 """
+
+
+def run_upgrade(setup_context):
+    """run upgrade to 6.0"""
+
+    setup_context.runImportStepFromProfile(
+        "profile-eea.api.dataconnector:default",
```

Comment:
@claudiaifrim @laszlocseh This upgrade step will re-import everything from the default profile types and it may override manual changes on production environments.

As you already registered a dedicated profile for this change, with this upgrade step, you should import typeinfo from it.

You'll need an entry in root `configure.zcml` like for 5.0 profile:

https://github.com/eea/eea.api.dataconnector/blob/develop/eea/api/dataconnector/profiles.zcml#L27-L33

And then the upgrade registry should look like:

https://github.com/eea/eea.api.dataconnector/blob/develop/eea/api/dataconnector/upgrades/configure.zcml#L44-L53

Thus, you can completely remove `to6.py` file and every code related to it.